src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -193,6 +194,14 @@ type StellarCoreCfg =
         t.Add("COMMANDS", logLevelCommands) |> ignore
         t.Add("CATCHUP_COMPLETE", self.catchupMode = CatchupComplete) |> ignore
 
+        // only latest images support Soroban configs
+        let latestCoreVersion (input: string) =
```

Comment:
Won't think break when we release v20?

Reply:
yeah, I guess I thought we'll have a proper upgrade mechanism by then. But I can add the 20.x.x version check just in case

Reply:
fixed